app/managers.py

```diff
@@ -0,0 +1,43 @@
+from models import Actor
+import sqlite3
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connect = sqlite3.connect("cinema.sqlite")
+        self.table_name = "actors"
```

Comment:
Ensure that the table schema is created if it doesn't exist. This is essential for the CRUD operations to function correctly.